go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/dunglas/frankenphp
 
-go 1.21
+go 1.22.0
```

Comment:
Just to add on to that, there are generally three types of PRs: feature, refactor, and bug-fix/update. Mixing these different types can create a lot of mental load, increase review time, make git-bisect not as useful, etc. I wrote an essay on it https://withinboredom.info/2024/07/16/classifying-pull-requests-enhancing-code-review-effectiveness/ but rereading it today, it could use some more polish. In any case, this change falls squarely in the refactor camp and doesn't seem required (maybe it is)?